tests/src/python/test_provider_postgres.py

```diff
@@ -397,7 +397,24 @@ def testTransactionNotDirty(self):
         ft1 = vl.getFeatures('pk=1')
         self.assertFalse(ft1.nextFeature(f))
 
-        p.setAutoTransaction(False)
+    def testTransactionTuple(self):
+        # create a vector ayer based on postgres
```

Comment:
s/ayer/layer/

Reply:
Thank you for pointing that out, it's fixed :)